test/e2e/setup.go

```diff
@@ -377,9 +391,34 @@ var _ = BeforeSuite(func() {
 
 	if err := setup(context.Background()); err != nil {
 		panic(err)
+	} else {
+		m := statsd.New(
+			context.Background(),
+			log.WithField("component", "e2emetrics"),
+			_env,
+			os.Getenv("MDM_E2E_ACCOUNT"),
+			os.Getenv("MDM_E2E_NAMESPACE"),
+			os.Getenv("MDM_E2E_STATSD_SOCKET"),
+		)
+		e2eHook := e2elogging.NewE2EHook(m)
+		log.Logger.AddHook(&e2eHook)
 	}
 })
 
 var _ = AfterSuite(func() {
 	log.Info("AfterSuite")
 })
+
+var _ = JustAfterEach(func() {
+	gtd := CurrentGinkgoTestDescription()
```

Comment:
What is the thinking behind using already deprecated `struct`?

Reply:
I was unaware of the deprecation - I will look into changing this.